clean_build_test.sh

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
-
-echo "CLEAN"
-./full_clean.sh
-
```

Comment:
I think it was cool to have this file even if its content only executes python commands. It is useful when people have problem with their local installation.

Reply:
so what, remove it?